core/src/main/java/com/onelogin/saml2/Auth.java

```diff
@@ -440,7 +441,7 @@ public void processSLO(Boolean keepLocalSession, String requestId) throws Except
 				logoutResponseBuilder.build(inResponseTo);
 				String samlLogoutResponse = logoutResponseBuilder.getEncodedLogoutResponse();
 
-				Map<String, String> parameters = new HashMap<String, String>();
+				Map<String, String> parameters = new LinkedHashMap<String, String>();
```

Comment:
`HashMap` ordering is non-deterministic and different across JDK7 and JDK8. Use a `LinkedHashMap` to make it easier to test.